arbitrary-image-stylization/package.json

```diff
@@ -0,0 +1,42 @@
+{
+    "name": "@magenta/style",
+    "version": "0.1.0",
+    "description": "Arbitrary image stylization in the browser.",
+    "main": "es5/index.js",
+    "types": "es5/index.d.ts",
+    "jsdelivr": "dist/magentastyle.js",
+    "unpkg": "dist/magentastyle.js",
+    "dependencies": {
+        "@tensorflow/tfjs": "^0.13.3"
+    },
+    "devDependencies": {
+        "@types/clone": "^0.1.30",
```

Comment:
Please remove packages you aren't using. I think this would include tape, clone, and fs.